integrated_channels/sap_success_factors/models.py

```diff
@@ -117,14 +119,17 @@ def get_learner_data_record(self, enterprise_enrollment, completed_date=None, gr
             completed_timestamp = parse_datetime_to_epoch(completed_date)
             course_completed = is_passing
 
-        return LearnerDataTransmissionAudit(
-            enterprise_course_enrollment_id=enterprise_enrollment.id,
-            sapsf_user_id=enterprise_enrollment.enterprise_customer_user.get_remote_id(),
-            course_id=enterprise_enrollment.course_id,
-            course_completed=course_completed,
-            completed_timestamp=completed_timestamp,
-            grade=grade,
-        )
+        sapsf_user_id = enterprise_enrollment.enterprise_customer_user.get_remote_id()
```

Comment:
As per the test instructions, just authenticating through SAPSF wasn't enough, as the transmission command would skip the user each time since an enrollment did not exist. So just a nit to update the test instructions to reflect that we need to be enrolled in something first.